backend/src/main/java/com/empire/World.java

```diff
@@ -358,24 +358,24 @@ public static World startNew(String gmPasswordHash, String obsPasswordHash, Map<
 		// Set religions for all regions to new owners.
 		for (Region r : w.regions) {
 			if (r.isSea()) continue;
-			if (r.kingdom == null) throw new RuntimeException(r.name + " is still unowned!");
-			if ("Unruled".equals(r.kingdom)) {
+			if (r.getKingdom() == null) throw new RuntimeException(r.name + " is still unowned!");
+			if ("Unruled".equals(r.getKingdom())) {
 				List<Ideology> possibilities = Ideology.getIdeologiesByReligion(r.culture.religion);
 				r.religion = possibilities.get((int)(Math.random() * possibilities.size()));
 			} else {
-				r.religion = nationSetup.get(r.kingdom).dominantIdeology;
+				r.religion = nationSetup.get(r.getKingdom()).dominantIdeology;
 			}
 		}
-		// Place shipyards. Want ~315 total (200?), split among all nations.
-		int shipyardsPerNation = (int)(Math.ceil(200.0 / nationSetup.keySet().size()));
+		// Place shipyards. Want ~80 total, split among all nations.
+		int shipyardsPerNation = (int)(Math.ceil(80.0 / nationSetup.keySet().size()));
 		for (String kingdom : nationSetup.keySet()) {
 			ArrayList<Region> regions = new ArrayList<>();
-			for (Region r : w.regions) if (kingdom.equals(r.kingdom) && r.isCoastal(w)) regions.add(r);
+			for (Region r : w.regions) if (kingdom.equals(r.getKingdom()) && r.isCoastal(w)) regions.add(r);
 			if (!regions.isEmpty()) {
 				for (int i = 0; i < shipyardsPerNation; i++) {
 					Construction c = new Construction();
 					c.type = "shipyard";
-					c.originalCost = 40;
+					c.originalCost = 80;
```

Comment:
Assuming this is intentional, the effect of this is simply that initial-state shipyards are razed at full cost?

Reply:
This is intentional. The cost factor is still applied, it's just that now those initial shipyards weren't assumed to be built at 50% off.

I forget the original motivation for saying that you only got half as much as usual for razing the initial shipyards. I think I was worried about the potential for nations that don't care about navies to gain a lot of gold by tearing down starting assets, and some concern about Industrial nations razing and rebuilding them for profit - but in the playtest this just surprised players who were expecting to get 40 gold for razing the shipyard but only got 20, and I think these concerns are foolish.